instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/__init__.py

```diff
@@ -43,11 +43,7 @@ def _instrument(self, **kwargs):
 
         # FIXME this is probably a pattern that will show up in the rest of the
         # ext. Find a better way of implementing this.
-        # FIXME Probably the evaluation of strings into boolean values can be
-        # built inside the Configuration class itself with the magic method
-        # __bool__
-
-        if Configuration().DJANGO_INSTRUMENT is False:
+        if environ.get("OTEL_PYTHON_DJANGO_INSTRUMENT") == "False":
```

Comment:
Does it have to be strict case sensitive comparison? Can this be something similar to this `environ.get("OTEL_PYTHON_DJANGO_INSTRUMENT", "").lower() == "false"`?

Reply:
We could, but that would be outside the scope of this PR.